namesrv/src/main/java/org/apache/rocketmq/namesrv/routeinfo/RouteInfoManager.java

```diff
@@ -52,14 +53,14 @@
     private final HashMap<String/* topic */, List<QueueData>> topicQueueTable;
     private final HashMap<String/* brokerName */, BrokerData> brokerAddrTable;
     private final HashMap<String/* clusterName */, Set<String/* brokerName */>> clusterAddrTable;
-    private final HashMap<String/* brokerAddr */, BrokerLiveInfo> brokerLiveTable;
+    private final Map<String/* brokerAddr */, BrokerLiveInfo> brokerLiveTable;
     private final HashMap<String/* brokerAddr */, List<String>/* Filter Server */> filterServerTable;
 
     public RouteInfoManager() {
         this.topicQueueTable = new HashMap<String, List<QueueData>>(1024);
         this.brokerAddrTable = new HashMap<String, BrokerData>(128);
         this.clusterAddrTable = new HashMap<String, Set<String>>(32);
-        this.brokerLiveTable = new HashMap<String, BrokerLiveInfo>(256);
```

Comment:
In the most situation, this hashmap was protected by a ReentrantReadWriteLock, so if we changed this map to ConcurrentHashmMap will cause less consistent with other codes, could we also use the lock to ensure thread safety？

Reply:
> In the most situation, this hashmap was protected by a ReentrantReadWriteLock, so if we changed this map to ConcurrentHashmMap will cause less consistent with other codes, could we also use the lock to ensure thread safety？

Change to use lock